# HG changeset patch
# User Dan Villiom Podlaski Christiansen <danchr@gmail.com>
# Date 1678122146 -3600
#      Mon Mar 06 18:02:26 2023 +0100
# Branch stable
# Node ID e6286ab9d66a0661fe5a34444be9621181f54530
# Parent  010a86744bfc6ede2e3ed3ecef3faee0de804f47
# EXP-Topic patches
hg

diff --git a/hg b/hg
--- a/hg
+++ b/hg
@@ -12,24 +12,6 @@ from __future__ import annotations
 import os
 import sys
 
-# Make `pip install --user ...` packages available to the official Windows
-# build.  Most py2 packaging installs directly into the system python
-# environment, so no changes are necessary for other platforms.  The Windows
-# py2 package uses py2exe, which lacks a `site` module.  Hardcode it according
-# to the documentation.
-if getattr(sys, 'frozen', None) == 'console_exe':
-    vi = sys.version_info
-    appdata = os.environ.get('APPDATA')
-    if appdata:
-        sys.path.append(
-            os.path.join(
-                appdata,
-                'Python',
-                'Python%d%d' % (vi[0], vi[1]),
-                'site-packages',
-            )
-        )
-
 try:
     from hgdemandimport import tracing
 except ImportError:
